### ISSUE TITLE
Error in computation of ORB, out of image limits

### ISSUE THREAD

**Issue by rmsalinas** [tum-vision/LDSO#15]
The FeatureDetector class is not considering the HALF_PATCH_SIZE border of ORB. Thus, I am experiencing segfaults in corners detected in the image borders.

**Comment by NikolausDemmel** [comment]
Hm... Probably the easiest way to address it is to limit the keypoint detector to return points far enough from the border that subsequent checks are not needed.

**Comment by rmsalinas** [comment]
Yes. So i did in my branch.El 29 ene. 2019 11:42, Nikolaus Demmel <notifications@github.com> escribió:Hm... Probably the easiest way to address it is to limit the keypoint detector to return points far enough from the border that subsequent checks are not needed.

**Comment by NikolausDemmel** [comment]
Could you please link the corresponding commit? Then I could cherry pick it. Thanks!

**Comment by rmsalinas** [comment]
https://github.com/rmsalinas/LDSOEl 29 ene. 2019 11:56, Nikolaus Demmel <notifications@github.com> escribió:Could you please link the corresponding commit? Then I could cherry pick it. Thanks!

**Comment by NikolausDemmel** [comment]
For future reference, I guess this is the diff @rmsalinas is referring to: https://github.com/rmsalinas/LDSO/compare/8d54820...01701747
